CoEpi/repo/SymptomRepo.swift

```diff
@@ -0,0 +1,30 @@
+import Foundation
+
+protocol SymptomRepo {
+    func symptoms() -> [Symptom]
```

Comment:
getSymptoms?

Reply:
Or `retrieveSymptoms`? To parallel with https://github.com/Co-Epi/app-ios/pull/9/files#diff-c91110676bccac94e01ef480f38875ecR7

Reply:
I'm tempted to change `retrieveContacts()` to be only `contacts()`. It's kind of obvious from the context, isn't it?